### PR TITLE
Update spaces-gpus.md

### PR DESCRIPTION
**Pull request by Michellehbn**
Per slack convo with @osanseviero, adding a section on Spaces Community GPU Grants. ❤️

### CODE REVIEW

**Review by julien-c — COMMENTED** [review]
nice! should we add a screenshot of the button, or not necessary?

File: docs/hub/spaces-gpus.md [hunk 1]
```diff
@@ -113,6 +113,10 @@ Additional lifecycle control over Spaces with upgraded hardware will soon be pro
 
 Additional information about billing can be found in the [dedicated Hub-wide section](./billing).
 
+### Community GPU Grants 
+
+Do you have an awesome Space but need help covering the GPU hardware upgrade costs? We love helping out those with an innovative Space so please feel free to apply for a community GPU grant and see if yours makes the cut! This application can be found in your Space hardware repo settings in the lower left corner under "sleep time settings".
```
Comment (osanseviero) [inline docs/hub/spaces-gpus.md:118]: Maybe I would change the wording in https://huggingface.co/docs/hub/spaces-overview#hardware-resources to be similar to the one here

**Review by osanseviero — APPROVED** [review]
Very cool! A screenshot would be nice!

File: docs/hub/spaces-gpus.md (hunk 1 above)
Comment (Michellehbn) [reply docs/hub/spaces-gpus.md:118]: Hi @osanseviero ! Okay sure thing, I'll open a new PR for this change too!

**Comment by Michellehbn** [comment]
Tadahhhh! Screenshot added! its good? 🤞

File: docs/hub/spaces-gpus.md [hunk 2]
```diff
@@ -113,6 +113,12 @@ Additional lifecycle control over Spaces with upgraded hardware will soon be pro
 
 Additional information about billing can be found in the [dedicated Hub-wide section](./billing).
 
+### Community GPU Grants 
+
+Do you have an awesome Space but need help covering the GPU hardware upgrade costs? We love helping out those with an innovative Space so please feel free to apply for a community GPU grant and see if yours makes the cut! This application can be found in your Space hardware repo settings in the lower left corner under "sleep time settings":
+
+![Community GPU Grant](https://user-images.githubusercontent.com/83347449/221907262-2260ecc6-c116-4757-a098-a8568f63e373.png)
```
Comment (SBrandeis) [inline docs/hub/spaces-gpus.md:120]:
```suggestion
![Community GPU Grant](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/ask-for-community-grant.png)
```

**Review by SBrandeis — APPROVED** [review]
LGTM, thank you!

File: docs/hub/spaces-gpus.md (hunk 1 above)
Comment (osanseviero) [reply docs/hub/spaces-gpus.md]: Thanks!

**Comment by Michellehbn** [comment]
Merging! (first merge ever! thanks @coyotte508 @SBrandeis)

**Pull request merged**